mysql-test/suite/innodb_undo/t/undo_tablespace.test

```diff
@@ -245,6 +245,33 @@ SELECT TABLESPACE_NAME, FILE_TYPE, FILE_NAME FROM INFORMATION_SCHEMA.FILES
        WHERE FILE_NAME LIKE '%undo%' ORDER BY TABLESPACE_NAME;
 DROP UNDO TABLESPACE undo_004;
 
+
+# Check that it won't work with LTFB
+LOCK TABLES FOR BACKUP;
+
+--error ER_CANT_EXECUTE_WITH_BACKUP_LOCK
+CREATE UNDO TABLESPACE undo_003 ADD DATAFILE 'undo3.ibu';
+
+--connect (con1,'localhost','root',,)
+SET @@session.lock_wait_timeout=1;
+
+--error ER_LOCK_WAIT_TIMEOUT
+CREATE UNDO TABLESPACE undo_003 ADD DATAFILE 'undo3.ibu';
+
+--connection default
+--disconnect con1
+
+UNLOCK TABLES;
+
+CREATE UNDO TABLESPACE undo_003 ADD DATAFILE 'undo3.ibu';
+ALTER UNDO TABLESPACE undo_003 SET INACTIVE;
+--let $inactive_undo_space = undo_003
+--source include/wait_until_undo_space_is_empty.inc
+DROP UNDO TABLESPACE undo_003;
+
+LOCK TABLES FOR BACKUP;
+UNLOCK TABLES;
```

Comment:
Didn't understand what this part is trying to do? Do you intend to
```
LOCK TABLES FOR BACKUP;
 --error ER_CANT_EXECUTE_WITH_BACKUP_LOCK 
DROP UNDO TABLESPACE undo_003;
UNLOCK TABLES;
DROP UNDO TABLESPACE undo_003;
```

Reply:
Also please add count_session and wait_until_count_session.inc

Reply:
we have to test ALTER UNDO as well

Reply:
I left it accidentally there, but it's a good idea, I added that part.

Reply:
:+1: